keanu-project/src/main/proto/io/improbable/keanu/BayesianNetwork.proto

```diff
@@ -7,30 +7,57 @@ option java_outer_classname = "KeanuSavedBayesNet";
 
 message BayesianNetwork {
     repeated Vertex vertices = 1;
-    repeated VertexValue defaultState = 2;
+    repeated StoredValue defaultState = 2;
+}
+
+message VertexID {
+    string id = 1;
```

Comment:
how come you've changed from `repeated int64` to `string`?

Reply:
Let's chat - but basically IDs are actually pretty bad and meaningless once saved (ie we can't use them for loading anyway).  They're just there to hook things together, so I didn't want to impose our internal structure (ie the long array) on to the save format - other generators can then use what they like.